Resources/views/ContentType/view_content_type_group.html.twig

```diff
@@ -36,7 +36,7 @@
                     {% for content_type in content_types %}
                         <tr>
                             <td>{{ content_type.id }}</td>
-                            <td><a href="{{ path('admin_contenttypeView', {'contentTypeId': content_type.id}) }}">{{ ez_trans_prop(content_type, "name") }}</a></td>
+                            <td><a href="{{ path('admin_contenttypeView', {'contentTypeId': content_type.id}) }}">{{ ez_trans_prop(content_type, "name", language_by_id[content_type.id]) }}</a></td>
```

Comment:
I might be wrong, but I don't think this is needed, [ez_trans_prop](https://doc.ez.no/display/TECHDOC/ez_trans_prop) will pick name by prioritized languages and fallback to main language on it's own afaik.